### PR TITLE
Triall acceptance - Files

### PR DESCRIPTION
**Pull request by ghost**
- Fixing the YAML indentation & formatting to pass EKS1.14 validation
- Specifying right Containers tags to deploy according to Triall Production running Version

### CODE REVIEW

File: helm/alfresco-content-services/templates/deployment-alfresco-adf-demo.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ spec:
       imagePullSecrets:
         - name: sphereonregistrydev
       containers:
-      - image: sphereonregistrydev.azurecr.io/alfresco/adf-demo:latest
+      - image: sphereonregistrydev.azurecr.io/alfresco/adf-demo:spms-415
```
Comment (LDMGN) [inline helm/alfresco-content-services/templates/deployment-alfresco-adf-demo.yaml]: Can we give this a better tag? A version number perhaps?

File: helm/alfresco-content-services/templates/deployment-filestore.yaml [hunk 2]
```diff
@@ -33,8 +33,9 @@ spec:
               # config map to use, defined in config-filestore.yaml
               name: {{ template "alfresco.shortname" . }}-filestore-configmap
           ports:
-            - containerPort: {{ .Values.filestore.image.internalPort }}
+          - containerPort: {{ .Values.filestore.image.internalPort }}
           resources:
+{{ toYaml .Values.filestore.resources | indent 12 }}
```
Comment (LDMGN) [inline helm/alfresco-content-services/templates/deployment-filestore.yaml:38]: I see that this was moved, but is this still the desired indentation?

Comment (ghost) [reply helm/alfresco-content-services/templates/deployment-filestore.yaml:38]: This is the one validated with EKS1.14 & being successfully deployed.
the command | indent 12 does match the indentation of resources:

File: helm/alfresco-content-services/templates/deployment-alfresco-adf-demo.yaml (hunk 1 above)
Comment (ghost) [reply helm/alfresco-content-services/templates/deployment-alfresco-adf-demo.yaml]: This is the tag of the Triall-prod container that we retrieve from the running production cluster.
This is relevant to the current setup and will need to become a parameter

File: helm/alfresco-content-services/templates/deployment-filestore.yaml (hunk 2 above)
Comment (LDMGN) [reply helm/alfresco-content-services/templates/deployment-filestore.yaml:38]: What is the reason for not indenting it already and removing `| indent 12`?

Comment (ghost) [reply helm/alfresco-content-services/templates/deployment-filestore.yaml:38]: Seems to be a syntax requested by Helm. 
The line was provided in the alfresco Template, I just moved it to the right place

Comment (LDMGN) [reply helm/alfresco-content-services/templates/deployment-filestore.yaml:38]: Discussed offline: The inserted values can be multiple lines, so the indentation needs to be inserted dynamicly.

**Review by LDMGN — APPROVED** [review]

**Pull request merged**